I don't understand the failling test and really don't know how this can be related to my changes. And I am not able to reproduce it on my local PC nor solve it.

```
self = <test_validation__property.TestValidateDetailDefault object at 0x7f25f51c7b20>
    def test_Bool(self) -> None:
        p = Bool()
        with pytest.raises(ValueError) as e:
            p.validate("junk")
>       assert matches(str(e.value), r"expected a value of type bool or bool_, got junk of type str")
E       AssertionError: assert False
E        +  where False = matches('expected a value of type bool or bool, got junk of type str', 'expected a value of type bool or bool_, got junk of type str')
E        +    where 'expected a value of type bool or bool, got junk of type str' = str(ValueError('expected a value of type bool or bool, got junk of type str'))
E        +      where ValueError('expected a value of type bool or bool, got junk of type str') = <ExceptionInfo ValueError('expected a value of type bool or bool, got junk of type str') tblen=2>.value
tests/unit/bokeh/core/property/test_validation__property.py:134: AssertionError
```

Maybe someone can explain or help.

--- a/docs/bokeh/source/conf.py
+++ b/docs/bokeh/source/conf.py
@@ -7,6 +7,7 @@
 
 # Standard library imports
 import os
+import re
 from datetime import date
 
 from sphinx.util import logging
@@ -175,7 +176,7 @@ if "BOKEH_DOCS_VERSION" in os.environ:
 else:
     json_url = "../switcher.json"
 if "dev" in version or "rc" in version:
-    version_match = f"dev-{version[:3]}"
+    version_match = "dev-" + re.match(r"\d\.\d+", version).group()
 else:
     version_match = version
 

--- a/tests/unit/bokeh/core/property/test_validation__property.py
+++ b/tests/unit/bokeh/core/property/test_validation__property.py
@@ -131,7 +131,7 @@ class TestValidateDetailDefault:
         p = Bool()
         with pytest.raises(ValueError) as e:
             p.validate("junk")
-        assert matches(str(e.value), r"expected a value of type bool or bool_, got junk of type str")
+        assert matches(str(e.value), r"expected a value of type bool or bool_?, got junk of type str")
     def test_Complex(self) -> None:
         p = Complex()
         with pytest.raises(ValueError) as e:

